var/spack/repos/builtin/packages/py-libensemble/package.py

```diff
@@ -44,7 +45,6 @@ class PyLibensemble(PythonPackage):
     variant("mpmath", default=False, description="Install with mpmath")
     variant("deap", default=False, description="Install with DEAP")
     variant("tasmanian", default=False, description="Install with tasmanian")
-    variant("pyyaml", default=False, description="Install with pyyaml")
 
     depends_on("py-setuptools", type="build")
```

Comment:
```suggestion
    depends_on("py-setuptools", type=("build", "run"))
```
It's listed under install_requires

Reply:
I think this is only needed for installation.

Reply:
You're right that it isn't needed after install. But it's listed under `install_requires`, so `pip check` will complain if it isn't found. The correct solution is to move it to `setup_requires`. But for now, it's a build/run dep.

Reply:
Added